docker-compose.yml

```diff
@@ -242,14 +245,17 @@ services:
         tag: "desec/rabbitmq"
     restart: unless-stopped
 
-  celery:
+  celery-email:
     build: api
     image: desec/dedyn-api:latest
     init: true
-    command: celery -A api worker -l info --uid nobody --gid nogroup
+    command: celery worker -A api -E -Q email_slow_lane,email_fast_lane,email_immediate_lane -c 8 -n email -l info --uid nobody --gid nogroup
     depends_on:
     - dbapi
     - nslord
+    - rabbitmq
+    volumes:
+    - zones:/zones:rw
```

Comment:
Does email celery have to write to zones?

Reply:
Currently not, but it runs on the code base that requires write-access to the zones. I do not see an immediate security problem here, so I opted to keep the write access enabled across all containers running the api image.